test/e2e/prometheus_test.go

```diff
@@ -3730,13 +3730,22 @@ func testPromMinReadySeconds(t *testing.T) {
 		t.Fatal("Creating prometheus failed: ", err)
 	}
 
-	promSS, err := kubeClient.AppsV1().StatefulSets(ns).Get(context.Background(), "prometheus-basic-prometheus", metav1.GetOptions{})
-	if err != nil {
-		t.Fatal(err)
-	}
+	var got int32
+	err = wait.Poll(time.Second, time.Minute*5, func() (bool, error) {
```

Comment:
I don't think that a wait.Poll is needed here since the sts should be created with the expected value.

Reply:
good point